edx_toggles/urls.py

```diff
@@ -3,10 +3,4 @@
 URLs for edx_toggles.
 """
 
-
-from django.conf.urls import url
-from django.views.generic import TemplateView
-
-urlpatterns = [
-    url('', TemplateView.as_view(template_name="edx_toggles/base.html")),
-]
+urlpatterns = []
```

Comment:
I don't actually know.  Is it ok to just delete this file?

Reply:
I'm not sure. I'd like to keep it to make it easier for other people to add urls there.